mlflow/pytorch/__init__.py

```diff
@@ -1099,3 +1127,36 @@ def print_auto_logged_info(r):
     autolog.__doc__ = autolog.__doc__.replace("MIN_REQ_VERSION", str(MIN_REQ_VERSION)).replace(
         "MAX_REQ_VERSION", str(MAX_REQ_VERSION)
     )
+
+
+def load_latest_checkpoint(model_class, run_id=None):
```

Comment:
Note:

No matter "save_weights_only" true or false, when loading back, it always requires the "model_class", and it does not need the model object.

This is different with Keras side API:
https://github.com/mlflow/mlflow/pull/10955/files#r1471374884

Reply:
makes sense to me!